tests/Tests/500__jupyterhub/test-versions.robot

```diff
@@ -94,3 +105,17 @@ Load Spawner Page
     [Documentation]    Suite Setup, loads JH Spawner
     Begin Web Test
     Launch JupyterHub Spawner From Dashboard
+
+Verify Python Version In Notebook
+    [Documentation]    Verifies python version >=3.8
+    ${python_version} =    Run Cell And Get Output    !python --version
+    ${python_version} =    Fetch From Right    ${python_version}    ${SPACE}
+    ${version_check} =    GTE    ${python_version}    3.8.0
+    Should Be Equal As Strings    ${version_check}    True
+
+Verify Errors In Logs
+    [Documentation]    Verifies that there are no errors in Logs
+    @{pods} =    Oc Get    kind=Pod    namespace=redhat-ods-applications
+    FOR    ${pod}    IN    @{pods}
+        ${logs} =    Oc Get Pod Logs    name=${pod['metadata']['name']}    namespace=redhat-ods-applications    container=${pod['spec']['containers'][0]['name']}
```

Comment:
How are you verifying that there are no errors in the logs?

Reply:
Incase if there are any errors with pod " Oc Get Pod Logs " is not returning logs and test fails

Reply:
I'm not sure it is true for all the pods and all the kind of errors. 

Maybe it'd be more clear if you write a statement to check the `$logs` variable and make the explicitly make the kw failing based on the statement result. Somethin like
```
IF not ${logs} is not empty THEN PASS
ELSE Fail
```

Reply:
What I  understood last Friday when we discussed about this is that you were looking at the logs manually and  checking for errors, as just searching the string "error" didn't work because there were false positives.

I think this is not an acceptable solution because the test should be 100% automatic. Maybe you can white list some of the "error" occurrences in the log or just have a list of errors to look for. Does the related Jira issue give you a clue of the messages to look for?

Reply:
@jgarciao No, the Jira Issue didn't  mention anything about Errors in logs.